apps/mobile/src/features/Settings/useCases/changeAppSettings/ApplicationSettingsScreenMock.tsx

```
@@ -0,0 +1,27 @@
+//     enable: React.useCallback(() => set(true), [set]),
+//     disable: React.useCallback(() => set(false), [set]),
+//   }
+// }
+
+export const mockUseCurrencyPairing = () => 'USD'
+
+export const mockIsAuthOsSupported = () => false
+
+export const mockUseAuthWithOs = ({onSuccess}: {onSuccess: () => void}) => ({
+  authWithOs: () => {
+    onSuccess: () => {
```

Biome (2.1.2)

[error] Do not use the onSuccess variable name as a label

The variable is declared here

Creating a label with the same name as an in-scope variable leads to confusion.

(lint/suspicious/noLabelVar)